test/scripts/drenv-selftest

```diff
@@ -6,6 +6,42 @@ cd $(dirname $(dirname $0))
 prefix="drenv-selftest-"
 env="envs/vm.yaml"
 
+verbose_mode=false
+
+usage() {
+ echo "Usage: $0 [OPTIONS]"
+ echo "Options:"
+ echo " -h, --help      Display this help message"
+ echo " -v, --verbose   Enable verbose mode"
+}
+
+handle_options() {
+  while [ $# -gt 0 ]; do
+    case $1 in
+      -h | --help)
+        usage
+        exit 0
+        ;;
+      -v | --verbose)
+        verbose_mode=true
+        ;;
+      *)
+        echo "Invalid option: $1" >&2
+        usage
+        exit 1
+        ;;
+    esac
+    shift
+  done
+}
+
+handle_options "$@"
+
+if [ "$verbose_mode" = true ]; then
+ echo "Verbose mode enabled."
+ set -v
+fi
```

Comment:
This may work, but I don't want to maintain options parsing in bash/sh. 

I think we can do much simpler change by passing all the options to drenv start/delete like this:

    drenv start --name-prefix $prefix $env "$@"

This add support for --verbose and any other option supported now or that will be added later to drenv.

Reply:
Done